src/reynir_correct/pattern.py

```diff
@@ -1547,15 +1667,17 @@ def plursub(self, kind: str, match: SimpleTree) -> None:
         for ann in self._ann:
             if ann.code in generic and ann.start == start and ann.end == end:
                 self._ann.remove(ann)
-
+        if suggest == so.tidy_text:
```

Comment:
Actually, it would be better to first set suggest to emulate_case(suggest, ...) (the way it is done in the parameters below), and then to compare it with so.tidy_text. (And then pass just suggest as a parameter, not call emulate_case() again.) Otherwise, a suggestion that only differs from the original text by being differently capitalized would be displayed, annoying users.

Reply:
Changed